plugins/scsi/fu-scsi-device.c

```diff
@@ -60,13 +60,27 @@ fu_scsi_device_probe(FuDevice *device, GError **error)
 	fu_device_add_instance_strsafe(device, "VEN", vendor);
 	fu_device_add_instance_strsafe(device, "DEV", name);
 	fu_device_add_instance_strsafe(device, "REV", version);
-	if (!fu_device_build_instance_id_quirk(device, NULL, "SCSI", "VEN", NULL))
+	if (!fu_device_build_instance_id_quirk(device, NULL, "SCSI", "VEN", NULL)) {
```

Comment:
Can we just do `fu_device_build_instance_id_quirk(device, error, "SCSI", "VEN", NULL)`?

Reply:
It's noisier, but yeah that works.